### DISCUSSION TITLE
Scrolling Vertically with 'Wrap Lines' Off

### DISCUSSION THREAD

**Discussion by shaunbharat** [community/community#10469] [Mobile] [answered]
Hey! I'm using GitHub on Android, and I find that scrolling through code vertically is a bit difficult when the 'wrap line' setting is off. However, the horizontal scrolling is way easier.

Mostly when I scroll vertically, it mistakes my swipes and scrolls horizontal instead.

This is a really annoying thing that happens, but I've never really paid any attention to it - I just try swiping as vertical as I can. But I noticed there was a feature request option in this app, so I thought I'd just suggest maybe fixing this.

Thanks!

Edit: forgot to add things (said below), and I meant when wrap lines is off.

So basically this happens when I'm scrolling vertically really quickly (flick swiping), and when I'm in the middle horizontally.

**Comment by brianlovin** [comment, marked as answer]
Thanks for the notes here @ShaunB56, I agree this should be less sensitive and have a better understanding of the direction you _intend_ to scroll. I'll pass this along to our team to see if we can make any progress on this.

**Reply by shaunbharat** [reply]
Thanks! I also had another suggestion that would help improve code browsing. So the idea is to add another option that allows you to scroll freely in any direction (when wrap lines is off). This means when scrolling, the direction doesn't 'hook' onto one specific direction.

Of course this should be an optional setting, and I would love to see this be a thing.
